### ISSUE TITLE
Does the Lighthouse user-agent have scrolling capability?

### ISSUE THREAD

**Issue by gnechita-toptal** [GoogleChrome/lighthouse#10326]
Hey everyone,

The tldr; is the question in the title of this issue, but I would like to offer more context to my question.

**Context:** 

We are using a js library to lazy load images and we think it might affect how TTI is reported by the Lighthouse tool. 

Since introducing that library and using it in the codebase we experienced a 2-second increase in TTI. Multiple changes were introduced in the codebase at the same time as the library, but we think the main culprit is this lazy loading library:  https://github.com/aFarkas/lazysizes

The part that is of interest in our particular situation is this: 

> lazysizes does not hide images/assets from Google. No matter what markup pattern you use. Google doesn't scroll/interact with your website. lazysizes detects, whether the user agent is capable to scroll and if not, reveals all images instantly.

So, does the Lighthouse User-Agent have scrolling ability?

**Comment by patrickhulce** [comment]
[That library only looks](https://github.com/aFarkas/lazysizes/blob/a2f37ec2371bff523ea8b94800682e4ec3596b9b/src/lazysizes-intersection.js#L174) at `'onscroll' in window` and checks for `Googlebot` in the UA, so yes from that library's perspective Lighthouse should have "scrolling ability". In practice, Lighthouse will not scroll or interact with the page specifically to ensure offscreen images aren't loaded before other high priority content.

**Comment by gnechita-toptal** [comment]
@patrickhulce thanks! 👍
